[sitecore-jss-rendering-host] Update webpack-dev-server dependency and retire devServer (#1424)
Sitecore/jss: CHANGELOG.md

## Unreleased

### 🛠 Breaking Changes

> * `[sitecore-jss-rendering-host]` devServer is retired. renderingHostServer is the only way to use rendering-host with react from now on.

Reviewer: ```suggestion
* `[sitecore-jss-rendering-host]` devServer is retired. renderingHostServer is the only way to start rendering host.
```
React it's not only one framework that we can be used

> ### 🎉 New Features & Improvements

Reviewer: Seems like changes not resolved as expected, below you can see same lines (check line 26), and 21.1.0 should be below "jss scaffold" feature
